spec/services/proofing/resolution/progressive_proofer_spec.rb

```diff
@@ -51,40 +49,63 @@
       )
     end
 
-    it 'returns a ResultAdjudicator' do
-      proofing_result = proof
+    context 'remote proofing' do
+      it 'returns a ResultAdjudicator' do
+        proofing_result = proof
 
-      expect(proofing_result).to be_an_instance_of(Proofing::Resolution::ResultAdjudicator)
-      expect(proofing_result.same_address_as_id).to eq(applicant_pii[:same_address_as_id])
-    end
+        expect(proofing_result).to be_an_instance_of(Proofing::Resolution::ResultAdjudicator)
+        expect(proofing_result.same_address_as_id).to eq(applicant_pii[:same_address_as_id])
```

Comment:
When I walk through the remote flow and add a log to the progressive proofer I see the applicant doesn't have `same_address_as_id` defined. So I think this spec would actually go in the ipp context. I think we should update the mock applicant we use for this spec with an applicant that doesn't have `same_address_as_id` defined. 
<img width="1253" alt="Screen Shot 2024-01-18 at 3 44 35 PM" src="https://github.com/18F/identity-idp/assets/20867088/0aabb4b5-90ff-4897-839f-22539a6d32bf">

Reply:
@svalexander Does the remote flow have `same_address_as_id` defined by default?

Reply:
no it looks like it's not a property that's available in the pii for the remote flow